libdevcrypto/Common.cpp

```diff
@@ -40,23 +41,18 @@ using namespace dev;
 using namespace dev::crypto;
 
 #ifdef ETH_HAVE_SECP256K1
-struct Secp256k1Context
-{
-	Secp256k1Context() { ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY); }
-	~Secp256k1Context() { secp256k1_context_destroy(ctx); }
-	secp256k1_context_t* ctx;
-	operator secp256k1_context_t const*() const { return ctx; }
-};
-static Secp256k1Context s_secp256k1;
+std::unique_ptr<secp256k1_context, void(*)(secp256k1_context*)> s_secp256k1(secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY), [](secp256k1_context* _c){ secp256k1_context_destroy(_c); });
 #endif
 
 static Secp256k1PP s_secp256k1pp;
 
+static h256 const c_curveN("0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141");
```

Comment:
I know that.

Can you add "FIXME" comment to move it back into the function after switch to VS 2015?